### PR TITLE
Small Bug Fixes

### PR DESCRIPTION
**Pull request by AppeazeTheCheese**
* Removed CSS that was explicitly setting the width of the FIRE popup, making it smaller than it should be.
* Copied `@match` and `@include` statements from FIRE (https://github.com/Charcoal-SE/userscripts/blob/master/fire/fire.user.js), so this script will run everywhere that FIRE does.
* Incremented version number.

### CODE REVIEW

**Review by double-beep — CHANGES_REQUESTED** [review]
Thanks for your PR! I intentionally chose to run the script only in Charcoal HQ because of all the requests made each time SD sends a message. If more rooms are to be added, then the data the userscript fetches from one room needs to be shared across different tabs, which is - IMO - unnecessary.

**Comment by AppeazeTheCheese** [comment]
Alright, that makes sense. I'll change it back to what it was with a few additions. I just found myself adding URLs in there more than once, so the last time, I just decided to copy everything from FIRE.

File: webpack.config.cjs [hunk 1]
```diff
@@ -27,8 +27,9 @@ module.exports = {
                      // @author       double-beep
                      // @contributor  Xnero
                      // @description  Watch, blacklist and see domain stats directly from the FIRE popup!
-                     // @match        https://chat.stackexchange.com/rooms/11540/charcoal-hq
-                     // @match        https://chat.stackexchange.com/transcript/11540*
+                     // @match        *://chat.stackexchange.com/rooms/11540/charcoal-hq?*
```
Comment (double-beep) [inline webpack.config.cjs:30]:
```suggestion
                     // @match        *://chat.stackexchange.com/rooms/11540/charcoal-hq*
```

Didn't see this earlier, why is `?` needed? It prevents the userscript from running in `https://chat.stackexchange.com/rooms/11540/charcoal-hq`.

**Review by double-beep — CHANGES_REQUESTED** [review]

**Comment by AppeazeTheCheese** [comment]
Ah, that is problematic. That's just a typo. I'll fix it really quick.

**Review by double-beep — APPROVED** [review]

**Pull request merged**